### PR TITLE
grosse contribution

### PR DESCRIPTION
**Pull request by Jauke**

### CODE REVIEW

**Comment by xvw** [comment]
Could you add a License ? (MIT) ?

File: src/package.rb [hunk 1]
```diff
@@ -24,8 +24,7 @@
     'EvEx.rb',
     'Commands.rb',
     'Incubator.rb',
-    'Tools.rb',
-    'Doc.rb'
```
Comment (xvw) [inline src/package.rb]: Are you sure ? If I remember, the "eval ingame" use the data of Doc.rb, no?

Comment (Jauke) [reply src/package.rb]: checked and ... he does not use it. :v

Comment (Jauke) [reply src/package.rb]: I re-check

Comment (Jauke) [reply src/package.rb]: I confirm. :)

**Comment by xvw** [comment]
ping @Grimimi  ?

File: RME.rb [hunk 2]
```diff
@@ -1,27 +1,49 @@
 # -*- coding: utf-8 -*-
-
 #==============================================================================
-# ** RME
+# ** RME v1.3.0
```
Comment (Jauke) [inline RME.rb]: *Finally a version for the releases! x)*

**Comment by Grimimi** [comment]
I have not followed recent changes in your stack. (My bad)
But the proposal seems cool!

**Pull request merged**